src/main/python/config/configuration.py

```diff
@@ -0,0 +1,9 @@
+from configparser import ConfigParser
```

Comment:
I do surely love me some `configparser` :pray:

Reply:
Best config manager I've ever seen.  Blows Java's out of the water.